pkg/pg/txdb_test.go

```diff
@@ -0,0 +1,63 @@
+package pg
+
+import (
+	"database/sql"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/utils/tests"
+)
+
+func TestTxDBDriver(t *testing.T) {
+	dbURL, ok := os.LookupEnv("CL_DATABASE_URL")
+	require.True(t, ok, "CL_DATABASE_URL must be set")
+	db := NewSqlxDB(t, dbURL)
```

Comment:
I've noticed weird behavior in my PR. If two tests use db and run sequentially, second fails with `conn closed` error.

Reply:
Ah, I think I know why that's happening. I changed txdb to use the test context for queries instead of `context.Background`, thinking that was an improvement over what's currently in `chainlink` repo.  But apparently that won't work--we'll have to put it back to `context.Background`, I guess there's just no context that would survive long enough for this.